README.md

```diff
@@ -1 +1,3 @@
-# fish-quest
\ No newline at end of file
+# fish-quest
+
+This is a quest about fish. And fishing for a quest.
```

Comment:
NB: can't wait to fish.

IDK what NB stands for, maybe Non-breaking, but comments with NB prefix don't need to be addressed before merging.